packages/pluggableWidgets/timeline-web/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "timeline",
+  "widgetName": "Timeline",
+  "version": "1.0.0",
+  "description": "Shows timeline",
+  "copyright": "Mendix BV",
+  "author": "Mehmet Ali IZCI",
+  "config": {
+    "widgetPath": "./tests/TestProjects/Mendix8/widgets",
+    "projectPath": "./tests/TestProjects/Mendix8",
+    "mendixHost": "http://localhost:8080",
+    "developmentPort": 3000
+  },
+  "packagePath": "com.mendix.widget.web",
+  "scripts": {
+    "start": "pluggable-widgets-tools start:server --open",
+    "dev": "pluggable-widgets-tools start:web",
+    "build": "pluggable-widgets-tools build:web",
+    "format": "pluggable-widgets-tools format",
+    "lint": "..\"/../../node_modules/.bin/eslint\" --config ../../../.eslintrc.js --ext .jsx,.js,.ts,.tsx src/",
+    "test": "..\"/../../node_modules/.bin/jest\" --config ../../../scripts/test/jest.web.config.js",
+    "test:e2e": "pluggable-widgets-tools test:e2e:web",
```

Comment:
You need to add now a `"pretest:e2e": "node ../../../scripts/test/updateAtlas.js",`

Reply:
To align with https://github.com/mendix/widgets-resources/pull/276

Reply:
And you need to add your project to a new branch in [testProjects](https://github.com/mendix/testProjects), remove svn configs from your `package.json`, and remove `widgetPath`.